packages/next/build/webpack/loaders/eslint-loader/base-rules.ts

```diff
@@ -0,0 +1,173 @@
+const baseRules = {
+  'array-callback-return': 'warn',
+  'default-case': ['warn', { commentPattern: '^no default$' }],
+  'dot-location': ['warn', 'property'],
+  eqeqeq: ['warn', 'smart'],
+  'new-parens': 'warn',
+  'no-array-constructor': 'warn',
+  'no-caller': 'warn',
+  'no-cond-assign': ['warn', 'except-parens'],
+  'no-const-assign': 'warn',
+  'no-control-regex': 'warn',
+  'no-delete-var': 'warn',
+  'no-dupe-args': 'warn',
+  'no-dupe-class-members': 'warn',
+  'no-dupe-keys': 'warn',
+  'no-duplicate-case': 'warn',
+  'no-empty-character-class': 'warn',
+  'no-empty-pattern': 'warn',
+  'no-eval': 'warn',
+  'no-ex-assign': 'warn',
+  'no-extend-native': 'warn',
+  'no-extra-bind': 'warn',
+  'no-extra-label': 'warn',
+  'no-fallthrough': 'warn',
+  'no-func-assign': 'warn',
+  'no-implied-eval': 'warn',
+  'no-invalid-regexp': 'warn',
+  'no-iterator': 'warn',
+  'no-label-var': 'warn',
+  'no-labels': ['warn', { allowLoop: true, allowSwitch: false }],
+  'no-lone-blocks': 'warn',
+  'no-loop-func': 'warn',
+  'no-mixed-operators': [
+    'warn',
+    {
+      groups: [
+        ['&', '|', '^', '~', '<<', '>>', '>>>'],
+        ['==', '!=', '===', '!==', '>', '>=', '<', '<='],
+        ['&&', '||'],
+        ['in', 'instanceof'],
+      ],
+      allowSamePrecedence: false,
+    },
+  ],
+  'no-multi-str': 'warn',
+  'no-native-reassign': 'warn',
+  'no-negated-in-lhs': 'warn',
+  'no-new-func': 'warn',
+  'no-new-object': 'warn',
+  'no-new-symbol': 'warn',
+  'no-new-wrappers': 'warn',
+  'no-obj-calls': 'warn',
+  'no-octal': 'warn',
+  'no-octal-escape': 'warn',
+  'no-redeclare': 'warn',
+  'no-regex-spaces': 'warn',
+  'no-restricted-syntax': ['warn', 'WithStatement'],
+  'no-script-url': 'warn',
+  'no-self-assign': 'warn',
+  'no-self-compare': 'warn',
+  'no-sequences': 'warn',
+  'no-shadow-restricted-names': 'warn',
+  'no-sparse-arrays': 'warn',
+  'no-template-curly-in-string': 'warn',
+  'no-this-before-super': 'warn',
+  'no-throw-literal': 'warn',
+  /**
+   * TS handles no-undef on its own.
+   * Enabling this would throw random errors at enums etc
+   */
+  'no-undef': 'error',
```

Comment:
> Does not break build for lint errors just logs them on the console

Is this behavior still desired, if so won't having these rules set to error cause the build to break?

Reply:
nice catch. changed the language. These rules are supposed to break the build